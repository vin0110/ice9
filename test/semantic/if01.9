if true -> 1; fi
