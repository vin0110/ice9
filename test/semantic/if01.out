IF:
  BLIT: TRUE [T_BOOL(1)]
(then)
  EXP:
    ILIT:1 [T_INT(1)]
(else)
  nil
